### ISSUE TITLE
Tabs: focus issues on keyboard tabbing when tabs are inside an md-dialog

### ISSUE THREAD

**Issue by pved2016** [angular/material#9039]
**Actual Behavior**:
-  `What is the issue? *` - When we use md-tabs inside an md-dialog focus on the default tab does not work on keyboard tabbing the first time. That is when we tab to the md-dialog with the keyboard the focus on the default tab is not visible the first time however when we continue to tab and return to the default tab the focus works the second and subsequent times.
-  `What is the expected behavior?` - focus on the default tab should always work for accessibility compliance

**CodePen** (or steps to reproduce the issue): *
-  `CodePen Demo which shows your issue:` Angular Material demo example - for Dialog - Tab Dialog (displays the same behavior)
-  `Details:`

**Angular Versions**: *
-  `Angular Version:`v1.4.9
-  `Angular Material Version:`v1.0.4-master-19966a3

**Additional Information**:
-  `Browser Type: *` Chrome, IE
-  `Browser Version: *`Version 11 for IE
-  `OS: *`Windows
-  `Stack Traces:`

---

Shortcut to create a [new CodePen Demo](http://codepen.io/team/AngularMaterial/pen/bEGJdd).
Note: `*` indicates required information. Without this information, your issue may be auto-closed.

> Do not modify the titles or questions. Simply add your responses to the ends of the questions.
>   Add more lines if needed.

**Comment by clshortfuse** [comment]
@pved2016 I'm having trouble reproducing this issue. I also tried on 

https://material.angularjs.org/1.0.4/demo/dialog
https://material.angularjs.org/1.0.5/demo/dialog

Maybe I'm not understanding the issue. When I press `Enter` on the `Tab Dialog` button, the dialog appears and my focus is moved to `More On Wikipedia` button on the first tab content. `Shift+Tab` moves up to the tab navigation bar and  `Tab`, moves to the next button.

**Comment by clshortfuse** [comment]
Actually, thanks to @topherfangio I can now see the issue. `ng-focus` seems to be redirecting focus improperly.

**Comment by clshortfuse** [comment]
I figured out the underlying issue and it actually isn't due to the dialog. A tab navigation item will not get focus styling if `lastClick` is false. `lastClick` only becomes true when you perform certain keypresses while focused on a nav item. This is why it doesn't get any style until the second iteration.

Essentially, the tab navigation items need to be "initialized" by a keypress. I have solved the issue by adding `ctrl.lastClick = false;` to the `redirectFocus()` function inside `tabsController.js` 

@topherfangio  Let's team up and double check this is a valid solution.

**Comment by manish2535** [comment]
I have a question. How are you supposed to read the content of the selected tab if there is no focusable element in the content? If there is no focusable element in the `<md-content>`, I cant see a way to jump into the `<md-content>` section to access it.
